.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<!--
+Thanks for submitting a Pull Request!
+
+Please shortly explain your contribution, and if fixing an issue from the tracker, please add a link to the issue.
+
+Be sure to go over each item in the list below before submitting your pull request.
+-->
+
+### Description
+
+Add your description here
+
+
+### Checklist
+
+- [ ] Did you change how any of these packages work? Added new functionality? Make sure to update the documentation
+- [ ] If you added a new package: make sure to add documentation for it
```

Comment:
I'd be a bit more specific on where to add documentation, something like:


Additionally, I'd add another bullet along the lines of:

```
- [ ] If you've added a new feature or new package, make sure to update the documentation at https://index.ros.org/doc/ros2/ describing the new feature.
```